script.js

```diff
@@ -0,0 +1,139 @@
+    },
+    {
+      name: "Mars",
+      src: "./img/MARS.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Jupiter",
+      src: "./img/JUPITER.jpg",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Saturn",
+      src: "./img/SATURN.png",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Uranus",
+      src: "./img/URAN.png",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+    {
+      name: "Neptune",
+      src: "./img/NEPTUNE.png",
+      text: "Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet."
+    },
+  ]
+
+
+  const bodyElement = document.getElementById("Body");
+
+  // TODO: write a simillar function to render navigation elements.
+
+  const MENUARRAY = [
+    {
+      name: "Home",
+      adres: "home.html"
+    },
+    {
+      name: "About",
+      adres: "about.html",
+    },
+    {
+      name: "Products",
+      adres: "products.html",
+    },
+    {
+      name: "Contact",
+      adres: "contact.html",
+    }
+  ]
+
+  const createMenu = function (elements) {
+    const ul = bodyElement.appendChild(document.createElement("ul"));
+      ul.className = "navigation";
+    elements.forEach(function (item) {
+        const li = ul.appendChild(document.createElement("li"));
+      li.className = "liClassName"
+     const text = li.appendChild(document.createElement("a"));
+      text.setAttribute('href', document.createTextNode(item.adres));
+      text.appendChild(document.createTextNode(item.name));
+    });
+  }
+
+  createMenu(MENUARRAY);
+
+  const renderMainContent = function (elements) {
+    elements.forEach(function (element) {
+      const divRow = bodyElement.appendChild(document.createElement("div"));
+      divRow.className ="DivRow";
+      const divImg = divRow.appendChild(document.createElement("div"));
+      divImg.className = "DivImg";
+      const img = divImg.appendChild(document.createElement("img"))
+      img.className = `Image BorderDiv ${element.name}`;
+      img.src = element.src;
+      const divTxt = divRow.appendChild(document.createElement("div"));
+      divTxt.className = "DivText";
+      const text = divTxt.appendChild(document.createElement("p"));
+      text.appendChild(document.createTextNode(element.text));
+      bodyElement.appendChild(document.createElement("br"));
+    });
+  }
+
+  renderMainContent(ELEMENTS);
```

Comment:
Also let's remove the space for every line in this file.

```suggestion
 renderMainContent(ELEMENTS);
```